src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color $transition-time;
+
+    &:hover {
+      color: #00acdc;
```

Comment:
if you put another color to a variable, put this one too so that it would not be offended